app/javascript/components/server-components/Library/Layout.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from "react";
```

Comment:
Please self-review your PR and explain why you had to create this legacy component here. It will help other reviewers understand the context of these changes in case the current reviewer is unavailable and other reviewers need to take it from here.